metricbeat/module/ceph/monitor_health/_meta/data.json

```diff
@@ -1,51 +0,0 @@
-{
```

Comment:
We still need / want this. To have one generated again you need to have a `docs.json` in the testdata directory.

Reply:
I recovered `data.json` original file with v2 events but I'm not fully understand how to work with `docs.json` in `testdata`. Do you mean to create an empty file there and the run with `-generate`? At least I tried this and nothing happened :confused:

Reply:
The docs.json must contain an even and will generate the `data.json` out of it when you run `-generate`.